docs/requirements.txt

```diff
@@ -2,4 +2,5 @@
 sphinx
 breathe
 recommonmark
-sphinx_material
+# TODO: Do we want to use the not-yet-released branch? Pin the commit?
+sphinx_immaterial @ git+https://github.com/jbms/sphinx-immaterial.git@merge-upstream
```

Comment:
To do.

Reply:
I've updated this to use a pinned commit from the `merge-upstream` branch since it seems to work fine. I'll keep an eye on the branch and try to update if/when it gets merged.